doc/amazon_sagemaker_operators_for_kubernetes_jobs.rst

```diff
@@ -30,7 +30,12 @@ Download the sample YAML file for training using the following command:
     wget https://github.com/aws/amazon-sagemaker-operator-for-k8s/blob/master/samples/xgboost-mnist-trainingjob.yaml
 
 Edit the \ ``xgboost-mnist-trainingjob.yaml`` file to replace the ``roleArn`` parameter with your \ ``<sagemaker-execution-role>``, and \ ``outputPath`` with your S3 bucket that the Amazon SageMaker
-execution role has write access to. Apply the YAML file using the
+execution role has write access to. The ``roleArn`` must have permissions so that Amazon SageMaker
```

Comment:
Does this need a `\` before the \`\`roleArn\`\` ? I'm not really sure how rust documentation works.